ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh

```diff
@@ -247,6 +248,18 @@ function initiate_upgrade {
 
     cp --remove-destination "$PGLIBNEW"/*.control "$PGSHARENEW/extension/"
     cp --remove-destination "$PGLIBNEW"/*.sql "$PGSHARENEW/extension/"
+
+    if [ -d "$PGLIBOLD" ]; then
+        WRAPPERS_LIB_PATH=$(find "$PGLIBNEW" -name "wrappers*so")
+        if [ -f "$WRAPPERS_LIB_PATH" ]; then
+            OLD_WRAPPER_LIB_PATH=$(find "$PGLIBOLD" -name "wrappers*so")
```

Comment:
the deb upgrades ensure that we won't match multiple files here? (not sure of behaviour of `-f` if multiple matches are passed in )